packages/ui/src/pages/index.tsx

```diff
@@ -56,6 +58,26 @@ import { useJourney } from '#/journeys/basic.tsx';
 import { JoyrideTooltip } from '#/components/joyride/tooltip.tsx';
 import { Preview } from '#/components/Preview.tsx';
 
+import customOrNodeUI from '#/components/customOrNodeUI.tsx';
+import { CUSTOM_OR_TYPE } from '#/components/customOrNode.tsx';
+import polineNodeUI from '#/components/polineNodeUI.tsx';
+import { type as POLINE_TYPE } from '#/components/polineNode.ts';
+
+//Register the custom node
+addCustomNodeTypes(customOrNodeUI)
```

Comment:
`addCustomNodeTypes` and `addPanelItem` functions need to be called before mounting the Editor